pkg/lib/infra/sms/ratelimit.go

```diff
@@ -16,10 +18,20 @@ type AntiSpamSMSBucketMaker struct {
 }
 
 func (m *AntiSpamSMSBucketMaker) MakeBucket(phone string) ratelimit.Bucket {
-	return ratelimit.Bucket{
-		Key:         fmt.Sprintf("sms-message:%s", phone),
-		Name:        "AntiSpamSMSBucket",
-		Size:        m.Config.Ratelimit.PerPhone.Size,
-		ResetPeriod: m.Config.Ratelimit.PerPhone.ResetPeriod.Duration(),
+	if m.Config.Ratelimit.PerPhone.Enabled {
```

Comment:
Do not make an infinite bucket. Check the enabled flag and determine whether we should take a token from the bucket.